DependencyInjection/FOSOAuthServerExtension.php

```diff
@@ -88,10 +88,12 @@ public function load(array $configs, ContainerBuilder $container)
             }
         }
 
-        if (!empty($config['authorize'])) {
-            $this->loadAuthorize($config['authorize'], $container, $loader);
+        if (empty($config['authorize'])) {
+            return;
         }
 
+        $this->loadAuthorize($config['authorize'], $container, $loader);
```

Comment:
Isn't this change doing the exact same thing?

Reply:
It's not the same thing as my version exit the function, so lines 99->104 won't be executed. In original version, they are executed even if authorize feature is disabled, which is a bug IMO.
Still waiting for the team to review & merge this PR

Reply:
Version 1.6 is about to be tagged soon. We may be able to squeeze this in.

I think you would have better chance of this getting thru if you solve the conflicts. Unit testing the change would be nice too.

Reply:
Conflicts solved, but this class is not unit tested

Reply:
You are right. I was mislead by the presence of the test file.